tex2pdf-tools/tex2pdf_tools/preflight/__init__.py

```diff
@@ -475,6 +475,9 @@ def collect_included_files(self, inc: list[str]) -> None:
         # inc[3] ... second argument (if present)
         # inc[4] ... third argument (if present)
         include_command = inc[0]
+        # special cases for environment style commands
+        if re.match(r"begin\s*{\s*overpic\s*}", include_command):
```

Comment:
I wonder - how does the test pass when the regex doesn't provide for optional argument [] brackets?